### PR TITLE
pam_tcb/Makefile: Move PAM_SO_SUFFIX to Make.defs.

### PR DESCRIPTION
**Pull request by besser82**
* pam_tcb/Makefile: Move PAM_SO_SUFFIX to Make.defs.
* Make.defs: Likewise.

### CODE REVIEW

**Comment by besser82** [comment]
@solardiz, this one is trivial and doesn't require any special review nor discussion.

**Review by ldv-alt — APPROVED** [review]

File: ChangeLog [hunk 1]
```diff
@@ -33,6 +33,11 @@
 	* tcb.spec: Remove -DENABLE_SETFSUGID.
 	* ci/run-build-and-tests.sh: Likewise.
 
+2021-11-17  Björn Esser  <besser82 at fedoraproject.org>
+
+	* pam_tcb/Makefile: Move PAM_SO_SUFFIX to Make.defs.
+	* Make.defs: Likewise.
+
```
[the remark this answers is not in the document]
Comment (besser82) [reply ChangeLog]: I've fixed the ChangeLog and commit to recent date.

**Review by ldv-alt — APPROVED** [review]

**Comment by besser82** [comment]
@solardiz Let's merge in the following order: this one, #11, #25, #24.  I'll rebase each PR on the main branch after the previous has been merged.

**Comment by solardiz** [comment]
> Let's merge in the following order: this one, #11, #25, #24. I'll rebase each PR on the main branch after the previous has been merged.

Sounds good. So now you need to rebase #11. Thank you!

**Comment by besser82** [comment]
> > Let's merge in the following order: this one, #11, #25, #24. I'll rebase each PR on the main branch after the previous has been merged.
> 
> Sounds good. So now you need to rebase #11. Thank you!

Done so!  You're welcome!  =)

**Comment by solardiz** [comment]
> Let's merge in the following order: this one, #11, #25, #24.

These are now merged. What's next?

**Comment by besser82** [comment]
> > Let's merge in the following order: this one, #11, #25, #24.
> 
> These are now merged. What's next?

When the remaining two PRs are merged, I'm fine with all I wantend to be in a v1.3 release; the PR about SELinux and the external helpers itself can wait (and is obviously big enough by itself for another future release).

**Comment by solardiz** [comment]
> When the remaining two PRs are merged, I'm fine with all I wantend to be in a v1.3 release; the PR about SELinux and the external helpers itself can wait (and is obviously big enough by itself for another future release).

Sounds good.

Where do you use tcb currently? Where will you use the 1.3 release? As-is or with patches added (such as in that pending SELinux and helpers PR)?

**Pull request merged**